### DISCUSSION TITLE
Allowing input when window is not focused?

### DISCUSSION THREAD

**Discussion by theh0mie** [project64/project64#2134] [Ideas]
Not sure if this was suggested already or has already been implemented, but even with the **Pause emulation when window is not active** checkbox off, the P64 window will not allow input if it is not in focus.

**Comment by toxeia** [comment]
I wish I could be more help than this, but with the "Pause emulation when window is not active" unchecked, my input is still taken while the window is not focused. Could it come down to your input device? If it's not something that can be exclusively controlled (like a controller) maybe it can't continue taking input when the window isn't focused?
